Applications/FileManager/FileUtils.cpp

```diff
@@ -9,6 +9,42 @@
 
 namespace FileUtils {
 
+int delete_directory(String directory, String& error_file)
```

Comment:
Would be nicer if the second argument's name was more descriptive, something like [`out_`]`file_there_was_problem_deleting`

Reply:
Maybe. The fact that it's a `String&` and not a `String` or `const String&` tells us that the function might write to it though. So the information is already here :)

Reply:
That said, the name `error_file` is indeed a bit ambiguous.

Reply:
Alright, I renamed if to `file_that_caused_error`.
It's surprisingly hard to come up with short and descriptive names.